build.gradle.kts

```diff
@@ -55,3 +55,10 @@ allprojects {
         mavenCentral()
     }
 }
+
+tasks.register("clean", Delete::class) {
```

Comment:
Could you please explain we do we need this? I used to have a `clean` task already.